### ISSUE TITLE
body is undefined in storage query resolution

### ISSUE THREAD

**Issue by attaboy** [ellipsis-ai/ellipsis-api#4]
when calling api.storage.query, the callback doesn't seem to receive the `body` parameter — it's undefined. A bug with the `request` library? Not sure what's going on here.

**Comment by attaboy** [comment]
I suspect it would be more useful if the query method returned the body directly in either case, so the user could expect the callback to receive something with a `data` property (and the optional `errors` property).

**Comment by andrewcatton** [comment]
What about if the server returns a 50x?

**Comment by attaboy** [comment]
I think (but not certain) I'd expect to include a .catch(…) to handle any errors outside of graphql.

**Comment by andrewcatton** [comment]
Yeah I think I agree with that.

**Comment by attaboy** [comment]
I think I figured out what's going with the original bug: the promise resolve callback only ever takes a single argument, according to the spec. So in the following code, only `response` will ever get passed to a callback — `body` is thrown away:

```
new Promise((resolve, reject) => {
…
  resolve(response, body);
})
```

**Comment by attaboy** [comment]
I made these changes in #3: all of the API methods now return the body directly, and the storage API rejects the promise if there's a non-200 response.

**Comment by attaboy** [comment]
Fixed by #3
